### DISCUSSION TITLE
TC-RMW tool - tangential and radial winds

### DISCUSSION THREAD

**Discussion by drmohanthota** [dtcenter/METplus#1999] [File I/O] [answered]
Hi I am using this TC-RMW tool in MET V10.0

I found that in the TCRMWconfig file some of the arguments are not listed, while they were present in the MET documentation guide (https://met.readthedocs.io/en/develop/Users_Guide/tc-rmw.html).
One of the argument is 
"The -data file_1 … file_n | data_file_list options specify the gridded data files or an ASCII file containing a list of files to be used."

Since I am new to this tool, I am wondering what kind of data files it needs in the input, is it required model forecast .nc files with U and V components in it? along with ATCF file.

**Comment by JohnHalleyGotway** [comment, marked as answer]
@drmohanthota I apologize for this issue question going unaddressed for so long.

Yes indeed, the `-data` command line option provides the gridded data files from which the model or analysis data should be extracted. These need to be in a format file format that MET can read, such as GRIB1, GRIB2, or a handful of NetCDF file flavors.

You can read about TC-RMW in [this chapter](https://met.readthedocs.io/en/latest/Users_Guide/tc-rmw.html#tc-rmw-tool) of the MET User's Guide. And if you're running METplus, please also see the [basic TCRMW use case](https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/TCRMW/TCRMW.html).

**Reply by JohnHalleyGotway** [reply]
@drmohanthota wondering if you have any followup questions about this discussion? If not, I'd like to mark it as being answered and lock it to prevent future posts. Of course, you can always create a new discussion when new issues or questions arise.
